extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -470,19 +556,38 @@ export class AmpStoryEmbeddedComponent {
   }
 
   /**
-   * Updates tooltip icon. If no icon src is declared, it sets a default src and
-   * hides it.
+   * Updates tooltip action icon. This is found on the right of the text.
+   * @param {!Object} embedConfig
+   * @private
+   */
+  updateTooltipActionIcon_(embedConfig) {
+    const actionIcon =
+      this.tooltip_.querySelector('.i-amphtml-tooltip-action-icon');
+
+    this.resources_.mutateElement(devAssert(actionIcon), () => {
+      Object.values(ActionIcon).map(iconClass => {
+        actionIcon.classList.toggle(iconClass, false);
+      });
```

Comment:
This loops twice on the array and does as many operations on classList that we have icons. Can we find a better solution, like maybe removing the class from `clearTooltip` earlier?